daemon/internal/newrelic/config/config.go

```diff
@@ -65,13 +65,13 @@ func NewDecoder(r io.Reader) *Decoder {
 //
 // Decode implements the following PEG.
 //
-// INI     = ((KEYWORD WS* '=' WS* VALUE) / COMMENT / WS)*
-// KEYWORD = ALPHA (ALPHA / NUMERIC / '_' / '.')*
-// VALUE   = QUOTED / DQUOTED / RAW
-// QUOTED  = '\'' .* '\''
-// DQUOTED = '"' .* '"'
-// RAW     = .* EOL
-// COMMENT = ('#' / ';') .* EOL
+//	INI     = ((KEYWORD WS* '=' WS* VALUE) / COMMENT / WS)*
+//	KEYWORD = ALPHA (ALPHA / NUMERIC / '_' / '.')*
+//	VALUE   = QUOTED / DQUOTED / RAW
+//	QUOTED  = '\'' .* '\''
+//	DQUOTED = '"' .* '"'
+//	RAW     = .* EOL
+//	COMMENT = ('#' / ';') .* EOL
```

Comment:
This needs to be indented with `tab` character in order to be treated as code to avoid `gofmt` replacing `''` - a pair of consecutive single quotes (' U+0027) - with Unicode right quote (” U+201D). See https://go.dev/doc/comment#syntax, Paragraph section for more details.